app/controllers/admin/overviews_controller.rb

```diff
@@ -62,8 +62,52 @@ def export_services
       end
     end
 
+    def export_dev_form_summary
+      respond_to do |format|
+        summary = service_summary('dev')
+
+        format.csv do
+          csv_data = CSV.generate do |csv|
+            csv << ['Service id', 'Service name', 'Locale', 'Start pages', 'Confirmation pages', 'Check your answers pages', 'Single Question pages', 'Multiple Question pages', 'Exit pages', 'Standalone pages', 'Address components', 'Autocomplete components', 'Checkbox components', 'Content components', 'Date components', 'Email components', 'Upload components', 'Multiupload components', 'Number components', 'Radio components', 'Text input components', 'Textarea components', 'Confirmation email enabled', 'Save and return enabled', 'Collect data via email', 'Send to JSON api', 'Receive csv', 'External start page enabled', 'Analytics enabled', 'Reference number enabled', 'Payment link enabled']
```

Comment:
This array of strings and the one a bit below seems identical to me? Maybe can be extracted to either a constant or a method to DRY hehe.